### PR TITLE
Update SLF4J versions to latest

### LINKED ISSUE #20379: Update SLF4J to latest version (1.7.36)
Liberty uses a large number of different versions of SLF4J and we should standardize on just one. Do not want to wait until there is a security issue before fixing this.

### PR DESCRIPTION
**Pull request by tevans78**
This is going to need some changes in WS-CD as well
#build

### CODE REVIEW

File: dev/io.openliberty.org.testcontainers/bnd.bnd [hunk 1]
```diff
@@ -28,8 +28,7 @@ Export-Package: \
 	com.sun.jna.*;version="5.8.0",\
 	org.apache.commons.*;version="1.20",\
 	org.testcontainers.*;version="${testContainerVersion}";-split-package:=merge-last,\
-	org.rnorth.*;version="1.0.8",\
-	org.slf4j.*;version="1.7.30";-split-package:=merge-last
```
Comment (KyleAure) [inline dev/io.openliberty.org.testcontainers/bnd.bnd:32]: If we remove this won't we also need to update each FAT suite that depends on TestContainers to include a dependency on the new SLF4J bundle?

Comment (tevans78) [reply dev/io.openliberty.org.testcontainers/bnd.bnd:32]: it isn't the job of the `io.openliberty.org.testcontainers` bundle to provide SLF4J packages. That is the job of the SLF4J bundles.

**Comment by tevans78** [comment]
These two PRs need to be delivered at the same time:
https://github.com/OpenLiberty/open-liberty/pull/20362
https://github.ibm.com/websphere/WS-CD-Open/pull/26314

This is a good combined PB:
https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_pAEbEKUtEeyLpORmV0nVgA

**Comment by wraschke** [comment]
Are the necessary files on DHE?

**Comment by tevans78** [comment]
> Are the necessary files on DHE?

None of these jars need to go in DHE. They are the versions from Maven Central.

**Review by cbridgha — APPROVED** [review]
Looks good - please open a followup issue to get this new version in our build from source repo, this is moving us backwards in this respect, and this remains important.

**Comment by tevans78** [comment]
#libby

**Pull request merged**